src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public DataProcessingException(String message) {
+    }
```

Comment:
The constructor `DataProcessingException(String message)` should call the superclass constructor with the message parameter. You can do this by adding `super(message);` inside the constructor body.